src/npc.cpp

```diff
@@ -1172,7 +1172,7 @@ void npc::make_angry()
         return; // We're already angry!
     }
 
-    add_msg( m_debug, "%s gets angry", name.c_str() );
+    add_msg( "%s gets angry", name.c_str() );
```

Comment:
I really missed that message in the chat log, so I added it. Should it be kept, by inclosing it in `_( )` function?